pkg/kwokctl/runtime/util.go

```diff
@@ -107,6 +108,30 @@ func ApplyComponentPatches(component *internalversion.Component, patches []inter
 	for _, patch := range patches {
 		applyComponentPatch(component, patch)
 	}
+	component.Args = sortArgsOnCommand(component.Args)
+}
+
+func sortArgsOnCommand(args []string) []string {
```

Comment:
https://github.com/kubernetes-sigs/kwok/pull/1187/files#diff-bf3d7bf2eb72d4d9cb11c9570611dc913d09cf53ebd4fd7ed1b365ae4bcdf16dR125-R129

It may have subcommands, so it can't use string sorting.